kythe-verification/testdata/basic/ImportsRef.hs

```diff
@@ -0,0 +1,7 @@
+-- Verify the references to the imported modules
+module Imports where
+
+-- - @"Data.Set" ref/imports vname("containers-0.5.7.1:Data.Set", "", "", "", "haskell")
```

Comment:
Note: Using the vname verification like below with verbatim package name might fail depending on the actual package name mangling. But ok for now, will see if causes any problems.